e2e/client.test.js

```diff
@@ -0,0 +1,451 @@
+import { expect, test } from '@playwright/test'
+const path = require('path')
+const multiTenant = require('../server/src/multiTenant')
+const algolia = require('../server/src/integrations/algolia')
+require('../server/scripts/algolia_settings/index')
+
+const createUser = `mutation CreateUser{
```

Comment:
Tu peux ajouter un commentaire pour dire à prettier de formatter le GraphQL à l'intérieur. A faire pour tous les morceaux de code GraphQL.

```suggestion
const createUser = /* GraphQL */ `mutation CreateUser{
```